config/development.json

```diff
@@ -42,7 +42,6 @@
 		"code-splitting": true,
 		"comments/filters-in-posts": true,
 		"comments/moderation-tools-in-posts": true,
-		"comments/management/comment-view": true,
```

Comment:
I think it's fine to enable this in other envs first, before removing all config checks. We can do that in a separate PR when things look OK. (Makes it super easy to turn off if we need to).

Reply:
+1